cfme/containers/provider/openshift.py

```diff
@@ -76,12 +82,20 @@ def href(self):
 
     @property
     def view_value_mapping(self):
-        return {
+
+        mapping = {
             'name': self.name,
-            'prov_type': 'OpenShift Container Platform',
             'zone': self.zone,
+            'metrics_type': self.metrics_type
         }
 
+        if appliance_is_downstream():
+            mapping['prov_type'] = 'OpenShift Container Platform'
+        else:
+            mapping['prov_type'] = 'OpenShift'
+
```

Comment:
**Optional**
Style - consider turning this into a ternary, though this example is 102 chars so it needs to be split

```
mapping['prov_type'] = 
    'OpenShift Container Platform' 
    if self.appliance.is_downstream 
    else 'OpenShift'